exercise_perseus_render/assets/src/vue/perseus.vue

```diff
@@ -457,6 +464,9 @@
     .paragraph
       padding: 4px
 
+    .fixed-to-responsive
+      display: inline-block
+
```

Comment:
> The content renderer will be more like a "viewport" into another app than an integrated experience with Kolibri.

We're literally going to be doing this with all the HTML5 apps. We definitely won't have control over the content styling that shows up inside the iframes.